README.md

```diff
@@ -86,16 +86,21 @@ You can specify these environment variables when starting the container using th
 $ docker run -e "MUMBLE_CONFIG_SERVER_PASSWORD=123"
 ```
 
+Note: In the unlikely case where a `<configName>` setting is unknown to the container, startup will fail. The root cause of this error is the fact that
```

Comment:
`This error` - what error? This note seems to be a bit out-of-context :eyes:

Reply:
added error